db/migrate/20241112143658_create_in_progress_email_match_logs.rb

```diff
@@ -0,0 +1,9 @@
+class CreateInProgressEmailMatchLogs < ActiveRecord::Migration[7.1]
```

Comment:
Yeah, it is storing the user id and the form id to use to validate if we have checked if the email addresses of the associated models are the same or different. So it will be saved when the email is the same and when it is different. But I didn't want to store the email addresses, just the internal ids since those aren't any type of PII. I do like having the word `Log` on the end since it really is only being used to log. My main concern with something like `InProgressEmailMatchesVaProfile` or `InProgressEmailMatchesVaProfileLog` is if it is too verbose. But it _does_ describe what it's doing without being too general.

Reply:
Yeah, I like `FormEmailMatchesProfileLog`. I think it more clearly describes the table and still isn't too awful long.